lib/riot-lol.js

```diff
@@ -45,6 +45,44 @@ class RiotLoL {
       throw err;
     }
   }
+
+  async getItems() {
+    try {
+      let response = await axios({
+        method: "get",
+        url: `${this.dataCdnUrl}/item.json`
+      });
+
+      let itemsMap = response.data.data;
+
+      return Object.keys(itemsMap).map(id => normalizeItem(itemsMap[id], id));
+    } catch (err) {
+      throw err;
+    }
+  }
+
+  async getItem({ item }) {
+    try {
+      let response = await axios({
+        method: "get",
+        url: `${this.dataCdnUrl}/item.json`
+      });
+
+      let itemsMap = response.data.data;
+
+      if (Object.keys(ITEMS_MAP).includes(item)) {
+        let itemId = ITEMS_MAP[item][0];
+        if (ITEMS_MAP[item].length === 1) {
+          return normalizeItem(itemsMap[itemId], itemId);
+        } else {
+          return ITEMS_MAP[item].map(id => normalizeItem(itemsMap[id], id));
+        }
+      }
+      return { error: "Item could not be found" };
```

Comment:
Instead of `return`, `throw` the error to send it to the catch block, which will then throw the error, which will be handled by the user's catch block.

```js
throw new Error('Item could not be found');
```